bedrock/newsletter/views.py

```diff
@@ -169,20 +170,13 @@ def updated(request):
     # Did they do an unsubscribe all?  then unsub=1 was passed
     unsubscribed_all = unsub == UNSUB_UNSUBSCRIBED_ALL
 
-    # Token might also have been passed (on remove_all only)
-    token = request.GET.get("token", None)
-    # token must be a UUID
-    if token is not None and not UUID_REGEX.match(token):
-        token = None
-
     # Say thank you unless we're saying something more specific
     if not unsub:
         messages.add_message(request, messages.INFO, thank_you)
 
     context = {
         "action": f"{settings.BASKET_URL}/news/custom_unsub_reason/",
         "unsubscribed_all": unsubscribed_all,
-        "token": token,
     }
     return l10n_utils.render(request, "newsletter/updated.html", context, ftl_files=FTL_FILES)
 
```

Comment:
It's your preference, but anywhere you have `DICT.get("key", None)`, you can drop the `None` since it is the default in Python to return `None` if the key is not found. Personally I let the default drive and leave the code a bit simpler and just do `DICT.get("key")`. Others have different preferences so this isn't a requested change from me.